src/main/java/uk/gov/hmcts/probate/transformer/CallbackResponseTransformer.java

```diff
@@ -368,7 +368,8 @@ private ResponseCaseDataBuilder getResponseCaseData(CaseDetails caseDetails, boo
                 .anyDeceasedChildrenDieBeforeDeceased(caseData.getAnyDeceasedChildrenDieBeforeDeceased())
                 .anyDeceasedGrandChildrenUnderEighteen(caseData.getAnyDeceasedGrandChildrenUnderEighteen())
                 .deceasedAnyChildren(caseData.getDeceasedAnyChildren())
-                .deceasedHasAssetsOutsideUK(caseData.getDeceasedHasAssetsOutsideUK());
+                .deceasedHasAssetsOutsideUK(caseData.getDeceasedHasAssetsOutsideUK())
+                .statementOfTruthDocument(caseData.getStatementOfTruthDocument());
```

Comment:
Is there a specific unit test for this?

Reply:
yes there is, it was hiding among other files. added now